src/unfold/admin.py

```diff
@@ -392,7 +415,7 @@ def _get_base_actions_submit_line(self) -> List[UnfoldAction]:
             self.get_unfold_action(action) for action in self.actions_submit_line or []
         ]
 
-    def get_custom_urls(self):
+    def get_custom_urls(self) -> Tuple:
```

Comment:
`Tuple[Tuple[str, str, notsure], ...]`

not sure is whatever is result of this:

`ClassBasedView.as_view()`

I guess `View`?